src/test-utils/testing-library.tsx

```diff
@@ -9,7 +9,7 @@ import { AutofillContextProvider } from '@box/metadata-editor';
 jest.unmock('react-intl');
 
 const Wrapper = ({ children }) => (
-    <AutofillContextProvider isAiSuggestionsFeatureEnabled={false}>
+    <AutofillContextProvider isAiSuggestionsFeatureEnabled={false} fetchSuggestions={() => Promise.resolve([])}>
```

Comment:
https://github.com/box/box-ui-elements/pull/3675/files#diff-d5567e5061ad3d70205e639ae3d61f96752bae7cf5f53e2da22999b78b1cc81bR12-R15

seems like there this PR is not being formatted on push, in this PR it seems to be formatting correctly, also these props are reversed. can we alphabetize these?

Reply:
I changed the order, the prop is assigned like this as a placeholder, it is currently worked on in other ticket (not the in in the linked PR), so I suggest to leave it as it is 👍